src/controls/QskSkinHintTableEditor.h

```diff
@@ -30,9 +30,11 @@ class QSK_EXPORT QskSkinHintTableEditor
     // generic access
 
     void setHint( QskAspect, const QVariant& );
+    void setHint( QskAspect, const QVariant&, QskAspect::State );
```

Comment:
Maybe the following would be more descriptive?

```suggestion
    void setHintForAllStates( QskAspect, const QVariant&, QskAspect::State );
```